examples/sprite_camera_follow/main.rs

```diff
@@ -0,0 +1,192 @@
+        sprite_sheet: sprite_sheet.clone(),
+        sprite_number: 1,
+        flip_horizontal: false,
+        flip_vertical: false,
+    };
+    world
+        .create_entity()
+        .with(transform)
+        .with(Player)
+        .with(sprite)
+        .with(Transparent)
+        .build()
+}
+
+fn init_camera(world: &mut World, parent: Entity) {
+    let mut transform = Transform::default();
+    transform.translation.z = 1.0;
+    world
+        .create_entity()
+        .with(Camera::from(Projection::orthographic(
+            -250.0, 250.0, 250.0, -250.0,
+        ))).with(Parent { entity: parent })
+        .with(transform)
+        .build();
+}
+
+struct Example;
+
+impl<'a, 'b> SimpleState<'a, 'b> for Example {
+    fn on_start(&mut self, data: StateData<GameData>) {
+        let world = data.world;
+        let resources_path = format!(
+            "{}/examples/sprite_camera_follow/resources",
+            application_root_dir()
+        );
+        let circle_sprite_sheet_handle = load_sprite_sheet(
+            world,
+            &format!("{}/Circle_Spritesheet.png", resources_path),
+            &format!("{}/Circle_Spritesheet.ron", resources_path),
+        );
+        let background_sprite_sheet_handle = load_sprite_sheet(
+            world,
+            &format!("{}/Background.png", resources_path),
+            &format!("{}/Background.ron", resources_path),
+        );
+
+        let _background = init_background_sprite(world, &background_sprite_sheet_handle);
+        let _reference = init_reference_sprite(world, &circle_sprite_sheet_handle);
+        let parent = init_player(world, &circle_sprite_sheet_handle);
+        init_camera(world, parent);
+    }
+}
+
+fn main() -> amethyst::Result<()> {
+    amethyst::start_logger(Default::default());
+
+    let root = format!("{}/examples/sprite_camera_follow", application_root_dir());
+    let path = format!("{}/resources/display_config.ron", root);
+
+    let config = DisplayConfig::load(&path);
+    let pipe = Pipeline::build().with_stage(
+        Stage::with_backbuffer()
+            .clear_target([0.1, 0.1, 0.1, 1.0], 1.0)
+            .with_pass(DrawSprite::new().with_transparency(
+                ColorMask::all(),
+                ALPHA,
+                Some(DepthMode::LessEqualWrite), // Tells the pipeline to respect sprite z-depth
+            )),
+    );
+
+    let input_path = format!("{}/resources/input.ron", root);
+    let game_data = GameDataBuilder::default()
+        .with_bundle(TransformBundle::new())?
+        .with_bundle(InputBundle::<String, String>::new().with_bindings_from_file(&input_path)?)?
+        .with(MovementSystem, "movement", &[])
+        .with_bundle(
+            RenderBundle::new(pipe, Some(config))
+                .with_sprite_sheet_processor()
+                .with_sprite_visibility_sorting(&[]), // Let's us use the `Transparent` component
+        )?;
+
+    let mut game = Application::build("./", Example)?.build(game_data)?;
```

Comment:
This would be where you set the resources path to `format!("{}/examples/sprite_camera_follow/resources", application_root_dir())` instead of the current `"./"`.

Reply:
I'm not sure anymore if we wanted to have examples use the relative path instead, since it's not good practice to hardcode paths to the compilation dir.